src/backend/distributed/sql/citus--9.5-1--10.0-1.sql

```diff
@@ -1,5 +1,6 @@
 -- citus--9.5-1--10.0-1
 
+DROP FUNCTION pg_catalog.upgrade_to_reference_table(regclass);
```

Comment:
would that make sense to add `IF EXISTS` so that in case it was already dropped, this won't error?